README.md

```diff
@@ -115,7 +115,7 @@ convict provides serveral predefined formats for validation that you can use ([u
 * `url`
 * `email`
 * `ipaddress` - IPv4 and IPv6 addresses
-* `duration` - miliseconds or a human readable string (e.g. "5 days")
+* `duration` - milliseconds or a human readable string (e.g. "3000", "5 days")
```

Comment:
@zaach @madarche should this actually say `(e.g. 3000, "5 days")`? This example gives milliseconds as a string :-\